zerver/lib/markdown/api_return_values_table_generator.py

```diff
@@ -141,3 +148,15 @@ def render_events(self, events_dict: Dict[str, Any]) -> List[str]:
         return text
 def makeExtension(*args: Any, **kwargs: str) -> MarkdownReturnValuesTableGenerator:
     return MarkdownReturnValuesTableGenerator(kwargs)
+
+def generate_data_type(schema: Mapping[str, Any]) -> str:
+    data_type = ""
+    if 'oneOf' in schema:
+        for item in schema['oneOf']:
+            data_type = data_type + generate_data_type(item) + " | "
+        data_type = data_type[:-3]
+    elif 'items' in schema:
+        data_type = "(" + generate_data_type(schema['items']) + ")[]"
+    else:
+        data_type = schema['type']
+    return data_type
```

Comment:
Is this function the same as the one you wrote for the request parameters?  If not, why not; and if so, can we just import the function here?

Reply:
@timabbott It was the same function;modified the code to import the function and use it.